CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## v1.1.0
+
 Added:
  * Add unstable `otel_metrics_ingestion` resource.
```

Comment:
nit:
```suggestion
 * Add unstable `chronosphere_otel_metrics_ingestion` resource.
```

in the customer-facing logs, we'll probably need to explain "unstable" differently